gsi/cert_utils/source/programs/grid-cert-info.in

```diff
@@ -230,66 +230,61 @@ while [ "X$1" != "X" ]; do
 done
 
 if [ "${rfc2253:-0}" != 1 ]; then
-    openssl_options="$openssl_options -nameopt rfc2253,-dn_rev"
+    openssl_options="$openssl_options -nameopt sep_multiline"
```

Comment:
You probably don't want to use the `sep_multiline` here as the default option, since then a `grid-cert-info` without options will also use it in printing the "-text" (i.e. `$toprint`) output. Instead you can add it just in (new) lines 269 and 280. The question is which is the best default format to use.